crates/networking/p2p/rlpx/connection/server.rs

```diff
@@ -282,9 +277,54 @@ impl GenServer for RLPxConnection {
                         &established_state.node,
```

Comment:
I'd remove this log warn as it will be logged anyway in the error handling section.